gradle/code-convention.yml

```diff
@@ -0,0 +1,15 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: org.opensearch.openrewrite.recipe.CodeCleanup
+displayName: CodeCleanup
+description: Automatically cleanup code, e.g. remove unnecessary parentheses, simplify expressions.
+recipeList:
+#  - org.openrewrite.java.RemoveUnusedImports
+#  - org.openrewrite.staticanalysis.RemoveUnusedLocalVariables
+#  - org.openrewrite.staticanalysis.RemoveUnusedPrivateFields
+  - org.openrewrite.staticanalysis.RemoveUnusedPrivateMethods
+#  - org.openrewrite.java.OrderImports
+#  - org.openrewrite.java.format.RemoveTrailingWhitespace
+#  - org.openrewrite.staticanalysis.EmptyBlock
+#  - org.openrewrite.staticanalysis.RemoveCallsToSystemGc
+#  - org.openrewrite.staticanalysis.UnnecessaryThrows
+#  - org.openrewrite.text.EndOfLineAtEndOfFile
```

Comment:
Can we move this to formatting.gradle like [removeUnusedImports](https://github.com/opensearch-project/OpenSearch/blob/main/gradle/formatting.gradle#L68)() we can have `removeUnusedPrivateMethods`?

Reply:
I guess I meant to say if we move `removeUnusedPrivateMethods` to formatting.gradle then we don't need `code-convention.yml` and other related files, right?

Reply:
sorry for confusion, yes of course can inline as well.

Reply:
is it better now?

Reply:
lets apply SOC on this for better overview. Thanks for guidance.